### ISSUE TITLE
run error

### ISSUE THREAD

**Issue by aerovr** [ngokevin/aframe-mobile-video-play-component#1]
error:Cannot find module '../node_modules/three-dev/examples/js/loaders/OBJLoader' from '/Users/yitao/labs/aframe-mobile-video-play-component/node_modules/aframe-core/lib', when use lib aframe-core.

**Comment by ngokevin** [comment]
Yeah, this component is pretty outdated (and probably not even useful). Might delete.
